### ISSUE TITLE
Can this project be made to work again?

### ISSUE THREAD

**Issue by bheeshmpita** [Antrikshy/reddit2Kindle#7]
I think no other project achieves the same functionality as this one as others skip the comment threads.
Can authorizing with user credentials help with that reddit api changes?
would love to see this project back again @Antrikshy

**Comment by Antrikshy** [comment]
Based on other contacts I've received in recent months, you may be thinking about this unrelated project: https://github.com/JamieMagee/reddit2kindle

**Comment by bheeshmpita** [comment]
Yes, i have gone through that project as well but all i am looking for is a tool that can convert reddit posts (including comments and images) in an ereader friendly format. Another project which i came across was [https://github.com/Ailothaen/RedditArchiver-standalone](url)
